packages/router/__tests__/router-test.ts

```diff
@@ -2269,6 +2277,10 @@ describe("a router", () => {
           "currentParams": {},
           "currentUrl": "http://localhost/",
           "defaultShouldRevalidate": true,
+          "formAction": "/fetch",
+          "formData": FormData {},
+          "formEncType": "application/x-www-form-urlencoded",
+          "formMethod": "post",
```

Comment:
Fetcher submission should be included to `shouldRevalidate` if the fetcher `action` redirected